include/motor_manager.h

```diff
@@ -8,6 +8,7 @@
 #include <iomanip>
 #include <chrono>
 #include <map>
+#include <poll.h>
```

Comment:
Not sure if this Linux file is ok to export in the header file. I assume it's ok because the library will only work on Linux as it currently is and you probably don't plan to support other platforms.

Reply:
It is in the cpp file though, why did you add it here?

Reply:
Nevermind I see pollfd.